x-pack/plugins/search_playground/common/models.ts

```diff
@@ -27,15 +27,27 @@ export const MODELS: ModelProvider[] = [
     provider: LLMs.openai,
   },
   {
-    name: 'Claude 3 Haiku',
+    name: 'Anthropic Claude 3 Haiku',
     model: 'anthropic.claude-3-haiku-20240307-v1:0',
     promptTokenLimit: 200000,
     provider: LLMs.bedrock,
   },
   {
-    name: 'Claude 3 Sonnet',
-    model: 'anthropic.claude-3-haiku-20240307-v1:0',
+    name: 'Anthropic Claude 3 Sonnet',
+    model: 'anthropic.claude-3-sonnet-20240229-v1:0',
```

Comment:
```suggestion
    name: 'Anthropic Claude 3.5 Sonnet',
    model: 'anthropic.claude-3-5-sonnet-20240620-v1:0',
```
Consider bumping this to the latest Sonnet model. This is the [default in the connector](https://github.com/elastic/kibana/pull/186671).